src/LogViewer/index.js

```diff
@@ -68,6 +68,7 @@ export default class LogViewer extends React.Component {
       this.followLog();
     }
 
+    parent.postMessage(parse(qs), '*');
```

Comment:
When using this against a logviewer that isn't embedded in an iframe, this will send a `postMessage` to itself. You should be able to guard this by checking if the parent is the same window:

```js
if (parent !== window) {
  parent.postMessage(parse(qs), '*');
}
```